Dockerfile

```diff
@@ -10,13 +10,17 @@ RUN apt-get update && \
     apt-get install curl --yes && \
     curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     apt-get update && \
-    apt-get install --yes build-essential python3-dev python3-pip git nodejs
+    apt-get install --yes build-essential python3-dev python3-pip
 
 # Import code, install code dependencies
-ADD . .
+ADD templates/ templates/
```

Comment:
I think you could do all this in one statement - https://stackoverflow.com/questions/30256386/how-to-copy-multiple-files-in-one-layer-using-a-dockerfile

(Consolidating layers in Docker helps with performance)

Reply:
This went badly. I retract this suggestion.